generated/niscope/niscope/session.py

```diff
@@ -3732,14 +3508,7 @@ def _get_equalization_filter_coefficients(self, number_of_coefficients):
                 property.
 
         '''
-        vi_ctype = _visatype.ViSession(self._vi)  # case S110
-        channel_ctype = ctypes.create_string_buffer(self._repeated_capability.encode(self._encoding))  # case C010
-        number_of_coefficients_ctype = _visatype.ViInt32(number_of_coefficients)  # case S210
-        coefficients_size = number_of_coefficients  # case B600
-        coefficients_ctype = get_ctypes_pointer_for_buffer(library_type=_visatype.ViReal64, size=coefficients_size)  # case B600
-        error_code = self._library.niScope_GetEqualizationFilterCoefficients(vi_ctype, channel_ctype, number_of_coefficients_ctype, coefficients_ctype)
-        errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
-        return [float(coefficients_ctype[i]) for i in range(number_of_coefficients_ctype.value)]
+        return self._library_interpreter.get_equalization_filter_coefficients(self._repeated_capability, number_of_coefficients)
 
     def _get_error(self):
```

Comment:
We don't seem to be calling this from anywhere. I believe this applies to the other drivers, as well.

Reply:
Stopped it from generating in session.py.